storage/innobase/handler/ha_innodb.cc

```diff
@@ -12476,6 +12476,69 @@ innodb_change_buffering_update(
 		 *static_cast<const char*const*>(save);
 }
 
+#ifdef UNIV_DEBUG
+static
+int
+innodb_track_changed_pages_validate(
```

Comment:
This and _update implement incorrect logic. The tracking subsys should never be initialised at runtime, the only allowed thing is to temporarily stop and restart it.